bundles/org.openhab.transform.jsonpath/NOTICE

```diff
@@ -11,3 +11,15 @@ https://www.eclipse.org/legal/epl-2.0/.
 == Source Code
 
 https://github.com/openhab/openhab2-addons
+
+== Third-party Content
```

Comment:
We are not bundling those libs within our bundle anymore, right? So if they are bundles that are installed separately, we do not need to declare any 3rd party content here.

Reply:
Yes, that is true. I will revert it.

Reply:
Done.